stack.yaml

```diff
@@ -2,6 +2,7 @@ resolver: lts-13.26
 pvp-bounds: upper
 packages:
 - '.'
+- lib/purescript-ast
```

Comment:
This naming convention follows what was [suggested](https://github.com/purescript/purescript/issues/3633#issuecomment-500745777). I don't have a strong feeling one way or the other. If we'd prefer a different convention, I'm down to rename.